### DISCUSSION TITLE
Color theme vars: digits

### DISCUSSION THREAD

**Discussion by bergser** [SchwarzIT/onyx#2195] [Q&A]
Hello, I've noticed that in https://github.com/SchwarzIT/onyx/blob/main/packages/sit-onyx/src/styles/variables/themes/digits-light.css

Primary colors for `digits` are:
```css
  --onyx-color-base-primary-100: var(--onyx-color-onyx-100);
  --onyx-color-base-primary-200: var(--onyx-color-onyx-200);
  --onyx-color-base-primary-300: var(--onyx-color-onyx-300);
  --onyx-color-base-primary-400: var(--onyx-color-onyx-400);
  --onyx-color-base-primary-500: var(--onyx-color-onyx-500);
  --onyx-color-base-primary-600: var(--onyx-color-onyx-600);
  --onyx-color-base-primary-700: var(--onyx-color-onyx-700);
  --onyx-color-base-primary-800: var(--onyx-color-onyx-800);
  --onyx-color-base-primary-900: var(--onyx-color-onyx-900);
```

but in https://github.com/SchwarzIT/onyx/blob/main/packages/sit-onyx/src/styles/variables/themes/value.css vars for `digits` are also existing

```css
  --onyx-color-digits-100: #e8fcfc;
  --onyx-color-digits-200: #bbeaed;
  --onyx-color-digits-300: #79dde2;
  --onyx-color-digits-400: #3dd0d8;
  --onyx-color-digits-500: #00c3cd;
  --onyx-color-digits-600: #00adb5;
  --onyx-color-digits-700: #00969d;
  --onyx-color-digits-800: #008085;
  --onyx-color-digits-900: #00696e;
  --onyx-color-digits-1000: #005356;
  --onyx-color-digits-1100: #003c3e;
  --onyx-color-digits-1200: #002626;
```

we want to use `digits` theme, but right now this is basically same as`onyx`

**Comment by larsrickert** [comment]
Hey @bergser, currently the onyx and digits are visually the same.
However, the default onyx theme might change in the future to use different colors. So if your projects is a digits project, we recommend to use the digits theme so you don't need to migrate potential future changes to the onyx theme.

You can refer to our [theming docs](https://onyx.schwarz/development/theming.html) for how to use the digits theme
